src/v3/test/integration/identify-with-password.test.tsx

```diff
@@ -50,7 +50,6 @@ describe('identify-with-password', () => {
             credentials: {
               passcode: 'fake-password',
             },
-            rememberMe: false,
```

Comment:
Any reason this was removed?

Reply:
ok to ignore it, backend will handle default value for optional field